File: benchmarks/accuracy/fixtures/python/dataflow_sql_injection.py
from flask import request
import sqlite3


def find_user():
    conn = sqlite3.connect("app.db")
    cursor = conn.cursor()
    username = request.args.get("username")
    query = f"SELECT id, email FROM users WHERE username = '{username}'"
    cursor.execute(query)
    return cursor.fetchone()

File: benchmarks/accuracy/fixtures/go/dataflow_sql_injection.go
package main

import (
	"database/sql"
	"fmt"
	"net/http"
)

func findUser(db *sql.DB, w http.ResponseWriter, r *http.Request) {
	name := r.URL.Query().Get("name")
	query := fmt.Sprintf("SELECT id, email FROM users WHERE name = '%s'", name)
	rows, err := db.Query(query)
	if err != nil {
		http.Error(w, "lookup failed", http.StatusInternalServerError)
		return
	}
	defer rows.Close()
}
